src/assets/global.css

```
@@ -1,2 +1,10 @@
 @import "tailwindcss";
 @import "@nuxt/ui";
+
+@theme {
+  --drawer-width: 256px;
+  --spacing-navbar: 1440px;
+}
+
+@layer utilities {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)